src/ServiceControl.Transports.ASBS/ServiceControl.Transports.ASBS.csproj

```diff
@@ -1,23 +1,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
A couple of these files have extra whitespace in them when makes it harder to see what has actually changed and will show up when looking at the history. Can we restore the original spacing?

Reply:
@mikeminutillo  you've got it. Will push an update shortly.